docs/usage/examples/self-hosting.md

```diff
@@ -396,3 +396,35 @@ The logging level output is controlled by the Bunyan logging library.
 |    40 | warn    |
 |    50 | error   |
 |    60 | fatal   |
+
+## Self-signed TLS/SSL certificates
```

Comment:
I'm happy with the content now.

One thing left to figure out: where do we put the new section? I'd like to organize our content so the most important stuff is at the top of the document, and the least important stuff at the bottom.

How often will our users need to use "Self-signed TLS/SSL certificates"?

Reply:
I think it is less important/common than the logging options above and therefore I put it at the end.

How often users really need is, depends highly on their system setup. If they work with thinks like github.com/gitlab.com and public registries, they don't need it at all as these use publicly trusted certificates.
If they run Renovate against their own internal systems (github.example.corp), these (might) have self-signed certificates, although nowadays many companies use even internally publicly trusted certificates.

I would say that, if you need this, you're most likely aware that your landscape uses self-signed certificates and you know what to look/search for. If you don't, you probably don't need that information at all and therefore I wouldn't put it too prominently.